website/source/docs/providers/aws/index.html.markdown

```diff
@@ -238,6 +230,14 @@ in excess of those allowed by the access policy of the role that is being assume
 
 Nested `endpoints` block supports the following:
 
+* `dynamodb_endpoint` - (Optional) Use this to override the default endpoint
+  URL constructed from the `region`. It's typically used to connect to
+  `dynamodb-local`.
+
+* `kinesis_endpoint` - (Optional) Use this to override the default endpoint
+  URL constructed from the `region`. It's typically used to connect to
+  `kinesalite`.
```

Comment:
I don’t have a working example to hand, but I’m pretty sure this is wrong. Note the error message says:

```
* provider.aws: "dynamodb_endpoint": [REMOVED] Use `dynamodb` inside `endpoints` block instead
* provider.aws: "kinesis_endpoint": [REMOVED] Use `kinesis` inside `endpoints` block instead
```

No `_endpoint`. [Code that parses the endpoints](https://github.com/hashicorp/terraform/blob/bcda5176ea8380929f54810ce98f7ea1f3713f12/builtin/providers/aws/provider.go#L545) block seems to omit the `_endpoint` as well.